install/kubernetes/cilium/Chart.yaml

```diff
@@ -1,9 +1,53 @@
 apiVersion: v2
 name: cilium
+displayName: Cilium
 home: https://cilium.io/
 version: 1.9.90
 appVersion: 1.9.90
-description: Helm chart for Cilium
+kubeVersion: ">= 1.12"
+icon: https://raw.githubusercontent.com/cilium/cilium/master/Documentation/images/logo-solo.svg
```

Comment:
seems not found for me :thinking:

Reply:
This one is only in the PR, so we would have to merge it first before it will work :)

Reply:
looks like the image is a part of this pr, so i guess it'll magically appear when it gets merged.

Reply:
oops my bad :face_with_thermometer:

Reply:
That's what I'm hoping at least :) I was following similar examples from other projects on artifact hub, I hope this is the step we need to make the logo show up over there.